ucsb-cs156-s24/proj-happycows-s24-4pm-5
# ProfitTable text look update following JobsTable

# OVERVIEW
In this PR, I updated the look of the profit table according to the JobsTable. 

- Timestamp has been changed to the same text format as the JobsTable
- Cows Health has been limited and rounded to 1 decimal point (originally was infinitely long, which is not good for visualization).
- Fixed PROFIT_PAGE_SIZE to 5 to help reconstructing the table
- Updated the ProfitTable so that the table shows only 5 rows on each page and added buttons to navigate between pages.

Closes https://github.com/ucsb-cs156-s24/proj-happycows-s24-4pm-5/issues/5#issue-2298914701
# SCREENSHOTS
![image](https://github.com/ucsb-cs156-s24/proj-happycows-s24-4pm-5/assets/108091023/e06cfe94-4b81-4ee5-8925-a863e9994863)


# TESTS

# Validation
1. Visit my dokku deployment here: https://happycows-baiminwang-dev.dokku-05.cs.ucsb.edu/
2. Login using an admin account
3. Navigate to "Admin" Tab bar and create a common if you don't have one yet.
4. Join a common and buy some cows.
5. Try to milk the cows, any times you want but preferably more than 10 times.
6. Then go back to the Commons, the profit table should by default show the 5 most recent histories and the page navigation buttons all work!
